etc/docker/README.md

```diff
@@ -0,0 +1,40 @@
+# OT BR Docker
+
+## Troubleshooting
```

Comment:
Instead of this header, can you say something like, "This file contains troubleshooting for common issues with OTBR Docker."